components/clinical/timeline/src/atoms/timeline-time.tsx

```diff
@@ -7,9 +7,33 @@ import {
   TimelineDomainResourceType,
 } from '@ltht-react/types'
 import { formatTime } from '@ltht-react/utils'
+import Icon from "@ltht-react/icon";
+import styled from "@emotion/styled";
 
-const TimelineTime: FC<Props> = ({ domainResource, domainResourceType, ...rest }) => {
+const StyledLeftTimeIcon = styled(Icon)`
+  cursor: pointer;
+  display: inline-block;
+  padding-right: 0.25rem;
+`
+
+const StyledRightTimeIcon = styled(Icon)`
+  cursor: pointer;
+  display: inline-block;
+  padding-left: 0.35rem;
+`
+
+const TimelineTime: FC<Props> = ({ domainResource, domainResourceType, orientation, pointInTimeClickHandler, ...rest }) => {
   if (!domainResource) return <></>
+  
+  const showPointInTime = orientation && pointInTimeClickHandler
+  const getTimeElement = (time: string) => { 
```

Comment:
This should be a component, probably called `TimeElement`  with appropriate props rather than a function that returns JSX. That'd also mean moving its definition out of the `Timeline` component so that React isn't re-declaring it on every `TimelineTime` render, so anywhere above line 25 or into it's own file.

Reply:
Haha, I did suspect that might be the case...